balancer-labs/frontend-v2
# Feature/cb 1click and change account

# Description

- Try to connect immediately with injected provider when user clicks "Connect Wallet" button
- Add "Change" button next to "Disconnect" in `AppNavSettings`, so user can change their wallet provider.

Misc.
- Use `ConnectorId` enum instead of just strings
- Removed references to Fortmatic wallet
- Cleanup `getConnectorName` and `getConnectorLogo` methods
- When getting provider from multiple injected providers, prefer MetaMask

[Original PR](https://github.com/balancer-labs/frontend-v2/pull/1933) (I couldn't push to that branch)

## Type of change


## How should this be tested?

1. Click any "Connect Wallet" button inside App.
2. If you are using Metamask (or similar provider), it should automatically connect with the App
3. If Metamask pops up and asks approval, Select Wallet modal should also open, so you can reject MM and select another Wallet
4. Once connected, open Account Menu
5. Clicking "Change" button inside account menu (pic below) should open Select Wallet modal so you can change your wallet

## Visual context

| New "Change" button |
| ----------------------|
|  ![Screen Shot 2022-06-22 at 14 10 20](https://user-images.githubusercontent.com/28311328/175021477-caa795ae-9d21-44f4-b7c8-919fc23de535.png)  |

## Checklist:
